main/getting-started/intro-zoe.md

```diff
@@ -137,81 +190,101 @@ const { outcome, payout: alicePayoutP } = await E(zoe).offer(
   alicePayments,
 );
 ```
-Zoe checks the invite for its validity for that contract instance. If it’s an invalid invite, the offer attempt fails, and Alice gets her refund in the payout. When she makes her offer, Alice receives an *outcome* and a *promise* that resolves to her *payout*. If the offer is valid, it's now an *active offer*.
-
-Now, Alice needs to get someone else involved who potentially will also make an offer, hopefully one that offers what she wants for the price she’s willing to pay for it. For the Atomic Swap contract, the *outcome* from her offer resolves to an *invite* she can send to others, in this case Bob. This is specific to this Atomic Swap contract; the outcome is whatever the contract instance returns from its hook that was attached to the invite when it was created:
-
-
+Zoe checks the invitation for its validity for that contract instance. If it’s an invalid invitation, the offer attempt fails, and Alice gets
```

Comment:
Zoe checks the invitation for its validity for that contract instance. When 
she makes her offer, Alice receives a (promise for) a `UserSeat` that lets
her monitor or control her offer, including getting payouts or offer results, 
or exiting it altogether. If her offer was invalid, the 
offer attempt fails, and Alice gets her refund in the payout. If the
offer is valid, it's now an *active offer*.

Now, Alice needs to get someone else involved who potentially will also 
make an offer, hopefully one that offers what she wants for
the price she’s willing to pay for it. The "offer result" is the object 
returned by the contract when it processes an offer. In the case of 
`atomicSwap`, it used `zcf.makeInvitation()` to make and return an *invitation* Alice can 
send to others, in this case Bob.